index.md

```diff
@@ -116,6 +116,15 @@ environments.
 
 See the [librpmem page](librpmem) for documentation and examples.
 
+#### libpmemset
+
+**libpmemset** aims to provide support for persistent file I/O operations,
+runtime mapping concatenation and multi-part support across poolsets.
+It relies on synchronous event streams for pool modifications.
+Its work is still in progress.
```

Comment:
its work is still in progress?

Reply:
I've added it because it didn't yet provide any functionality.
But maybe it's better without this fragment.